### ISSUE TITLE
OBS doesn't recognise obs-ndi

### ISSUE THREAD

**Issue by Luuk3333** [Palakis/obs-ndi#227]
Hey,

I can't get obs-ndi to load and the log file doesn't help me either. I'm on KDE neon and compiled OBS myself in order to enable NVENC. Afterwards I installed both `libndi3_3.5.1-1_amd64.deb
` and `obs-ndi_4.5.2-1_amd64.deb` using `dpkg -i`.
For some reason OBS just doesn't seem to recognise obs-ndi. I saw other log files start some lines with `[obs-ndi]` but my log lacks those completely.
If I install OBS using the PPA and install obs-ndi it works fine, but then I lack NVENC.

OBS log:
```
$ obs --verbose
Attempted path: share/obs/obs-studio/locale/en-US.ini
Attempted path: /usr/share/obs/obs-studio/locale/en-US.ini
Attempted path: share/obs/obs-studio/themes/Dark.qss
Attempted path: /usr/share/obs/obs-studio/themes/Dark.qss
info: Command Line Arguments: --verbose
info: CPU Name: Intel(R) Core(TM) i5-7200U CPU @ 2.50GHz
info: CPU Speed: 2736.013MHz
info: Physical Cores: 2, Logical Cores: 4
info: Physical Memory: 7760MB Total, 4661MB Free
info: Kernel Version: Linux 4.15.0-34-generic
info: Distribution: "KDE neon" "16.04"
info: Window System: X11.0, Vendor: The X.Org Foundation, Version: 1.19.6
info: Portable mode: false
QMetaObject::connectSlotsByName: No matching signal for on_advAudioProps_clicked()
QMetaObject::connectSlotsByName: No matching signal for on_advAudioProps_destroyed()
QMetaObject::connectSlotsByName: No matching signal for on_program_customContextMenuRequested(QPoint)
info: OBS 22.0.2-140-g47285c2 (linux)
info: ---------------------------------
info: ---------------------------------
info: audio settings reset:
        samples per sec: 44100
        speakers:        2
info: ---------------------------------
info: Initializing OpenGL...
info: Loading up OpenGL on adapter NVIDIA Corporation GeForce 940MX/PCIe/SSE2
info: OpenGL loaded successfully, version 3.2.0 NVIDIA 384.130, shading language 1.50 NVIDIA via Cg compiler
info: ---------------------------------
info: video settings reset:
        base resolution:   1920x1080
        output resolution: 1920x1080
        downscale filter:  Bilinear
        fps:               60/1
        format:            NV12
        YUV mode:          601/Partial
info: Audio monitoring device:
        name: Default
        id: default
info: ---------------------------------
libDeckLinkAPI.so: cannot open shared object file: No such file or directory
info: No blackmagic support
info: NVENC supported
info: VLC found, VLC video source enabled
info: ---------------------------------
info:   Loaded Modules:
info:     vlc-video.so
info:     text-freetype2.so
info:     rtmp-services.so
info:     obs-x264.so
info:     obs-transitions.so
info:     obs-outputs.so
info:     obs-libfdk.so
info:     obs-filters.so
info:     obs-ffmpeg.so
info:     linux-v4l2.so
info:     linux-pulseaudio.so
info:     linux-jack.so
info:     linux-decklink.so
info:     linux-capture.so
info:     linux-alsa.so
info:     image-source.so
info:     frontend-tools.so
info: ---------------------------------
info: ==== Startup complete ===============================================
info: All scene data cleared
info: ------------------------------------------------
info: pulse-input: Server name: 'pulseaudio 8.0'
info: pulse-input: Audio format: s16le, 44100 Hz, 2 channels
info: pulse-input: Started recording from 'alsa_output.pci-0000_00_1f.3.analog-stereo.monitor'
info: pulse-input: Server name: 'pulseaudio 8.0'
info: pulse-input: Audio format: s16le, 44100 Hz, 2 channels
info: pulse-input: Started recording from 'alsa_input.pci-0000_00_1f.3.analog-stereo'
info: xcompcap: [window-capture: 'Window Capture (Xcomposite)'] update settings:
        title: Desktop  Plasma
        class: plasmashell
        Has alpha: yes
        Found exact GLXFBConfig: yes
info: Switched to scene 'Scene 2'
info: ------------------------------------------------
info: Loaded scenes:
info: - scene 'Blackout':
info: - scene 'Screen capture':
info:     - source: 'Window Capture (Xcomposite)' (xcomposite_input)
info: - scene 'Scene 2':
info: ------------------------------------------------
info: adding 23 milliseconds of audio buffering, total audio buffering is now 23 milliseconds
info: adding 23 milliseconds of audio buffering, total audio buffering is now 46 milliseconds

```

Any help would be appreciated.
Regards,
Luuk

**Comment by matiaspl** [comment]
I think your compiled OBS gets installed to /usr/local/share whereas
obs-ndi deb package installs itself to /usr/share. Try moving the plugin
files to /usr/local/share obs-studio folder.

pon., 24.09.2018, 19:02 użytkownik Luuk <notifications@github.com> napisał:

**Comment by Luuk3333** [comment]
Thanks for your reply.

I tried moving the plugins but got the same result.
Both OBS and obs-ndi are installed in `/usr/share/obs`, checked with `dpkg -L`.  
To check which folder OBS uses, I renamed the folder `obs-x264` to `obx-x264_` in both obs-plugins folder. In the `/usr/share/obs/obs-plugins` folder I got: `warning: Failed to load 'en-US' text for module: 'obs-x264.so'`. So obs-ndi is in the correct folder.
